src/components/Content/Content.tsx

```diff
@@ -55,42 +56,75 @@ class Content extends React.Component<any, any> {
       styles: stylesProp = {},
       id,
       emptyMessage,
-      effects: { saveChanges, setItem },
+      effects: { saveChanges, setItem, deleteItem, stageChange },
       state: { thing: { item, valid } },
       type,
       history,
     } = this.props;
 
-    const { creating, editing, saving } = this.state;
+    const { creating, editing, saving, disabling } = this.state;
     return (
       <div className={`content ${css(styles.container, stylesProp)}`}>
         <ControlContainer style={styles.controls}>
           {!editing &&
             !creating && (
-              <div>
-                <Button
-                  basic
-                  color="green"
-                  onClick={async () => {
-                    await setItem(null, type);
-                    this.setState({ creating: true, editing: false });
-                  }}
-                  size="tiny"
-                  style={{ fontWeight: 'bold' }}
-                >
-                  Create
-                </Button>
-                {id && (
+              <Aux>
+                <div>
                   <Button
-                    color="blue"
-                    onClick={() => this.setState({ editing: true, creating: false })}
+                    basic
+                    color="green"
+                    onClick={async () => {
+                      await setItem(null, type);
+                      this.setState({ creating: true, editing: false });
+                    }}
                     size="tiny"
-                    style={{ fontWeight: 'normal' }}
+                    style={{ fontWeight: 'bold' }}
                   >
-                    Edit
+                    Create
+                  </Button>
+                  {id && (
+                    <Button
+                      color="blue"
+                      onClick={() => this.setState({ editing: true, creating: false })}
+                      size="tiny"
+                      style={{ fontWeight: 'normal' }}
+                    >
+                      Edit
+                    </Button>
+                  )}
+                </div>
+                {RESOURCE_MAP[type].noDelete ? (
+                  <Button
+                    basic
+                    disabled={disabling || (item || {}).status === 'Disabled'}
+                    loading={disabling}
+                    onClick={async () => {
+                      this.setState({ disabling: true });
+                      await stageChange({ status: 'Disabled' });
+                      await saveChanges();
+                      this.setState({ ...INITIAL_STATE });
+                    }}
+                    size="tiny"
+                    color="red"
+                    style={{ fontWeight: 'bold' }}
+                  >
+                    Disable
+                  </Button>
+                ) : (
```

Comment:
How do you re-enable?

Reply:
right now manually edit the status. Not sure what we could do as a one click action, go to pending?
Pending -> Disabled -> Pending
Approved -> Disabled -> Pending

Reply:
Good point, might make sense to have 'disabled' as a boolean flag in the DB instead of being under 'status'
Let's leave it for now then